docs/shared/ungrib.yaml

```
@@ -0,0 +1,13 @@
+ungrib:
+  execution:
+    mpicmd: srun
+    batchargs:
+      nodes: 1
+      walltime: 00:05:00
+    executable: "path/to/ungrib.exe"
+  gfs_file: "/path/to/dir/gfs.2024.03.18.12"
```

```suggestion
  gfs_file: "/path/to/dir/gfs.t12z.pgrb2.0p25.f000"
```
Readers might better recognize the filename that NCEP uses, which in this case would be `gfs.t12z.pgrb2.0p25.f000`, than the name we used during development and testing.

(FWIW, NCEP only encodes the hour part of the cycle information (`t12z` for the 12 UTC cycle) in the filename; the year/month/day information is encoded in their directory structure. The GRIB data inside the file contains cycle information, too, though, so it's not as if paths and filenames are the only way to get that.)